client/gutenberg/extensions/subscriptions/save.jsx

```diff
@@ -0,0 +1,14 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { RawHTML } from '@wordpress/element';
+
+export default function Save( { attributes } ) {
+	const { show_subscribers_total } = attributes;
+	return (
+		<RawHTML
+		>{ `[jetpack_subscription_form show_subscribers_total="${ show_subscribers_total }" show_only_email_and_button="true"]` }</RawHTML>
```

Comment:
On 2nd thought and after slack convo:

- we'd still need `<RawHTML>` — I don't exactly remember what are implications of leaving it out (perhaps outputting extra `<p>`'s?) but we had the same situation with Simple payments block outputting a shortcode: 

    https://github.com/Automattic/wp-calypso/blob/51bed961432753c434b0a31efc65033843c201b2/client/gutenberg/extensions/simple-payments/save.js#L10

- Since the shortcode is quite simple, it makes sense to keep outputting it as a simple string. Using `shortcode` from Gutenberg would make sense with a whole lot of attributes.